@google-cloud/storage/bucket/index.d.ts

```diff
@@ -0,0 +1,102 @@
+/// <reference types="node" />
+
+import { ReadStream } from "fs";
+import { Channel, ChannelConfig } from "../channel";
+import { File } from "../file";
+import {
+  Acl,
+  ApiResponse,
+  UploadOptions
+} from "../storage";
+
+declare module "@google-cloud/storage/bucket" {
```

Comment:
@yuit Honestly, I was just following the contribution guidelines for DT. I can change this to simply export the class declarations and interfaces.